docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -554,9 +554,114 @@ Payload objects with the following scopes take the form that follows:
 
 Consult the [pages section of the Channel Webhooks Guide](/docs/integrations/webhooks/events/channels#pages).
 
+## Price Lists
+The following price lists webhook events fire in response to actions that affect a specific price list:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceList/created`  | Fires when a price list is created. | 
+| `store/priceList/updated/` | Fires when a price list is updated. | 
+| `store/priceList/activated/` | Fires when a price list is activated. | 
+| `store/priceList/deactivated/` | Fires when a price list is deactivated. | 
+| `store/priceLists/deleted/` | Fires when a price list is deleted. | 
+
+Payload objects with the following scopes take the form that follows:
+
+* `store/priceList/created`
+* `store/priceList/updated/` 
+* `store/priceList/activated/`
+* `store/priceList/deactivated/`
+* `store/priceLists/deleted/`
+
+
+```json filename="Example price list created payload object" showLineNumbers copy
+{
+  "scope": "store/priceList/created",
+  "store_id": "1001197568",
+  "data": {
+    "price_list_id": 5
+  },
+  "hash": "e0c298b8097a6a2f39d17e593a9b360f5b2fef7d",
+  "created_at": 1709327895,
+  "producer": "stores/{store_hash}"
+}
+```
+```json filename="Example price lists deleted payload object" showLineNumbers copy
+{
+  "scope": "store/priceLists/deleted",
+  "store_id": "1001197568",
+  "data": {
+    "price_list_ids": [
+      5
+    ]
+  },
+  "hash": "e0c298b8097a6a2f39d17e593a9b360f5b2fef7d",
+  "created_at": 1709327895,
+  "producer": "stores/{store_hash}"
+}
+```
+
+
+## Price list records
+
+The following price lists records webhook events fire in response to actions that affect a specific price list record:
+
+| Name / Scope | Description |
+|:-------------|:------------|
+| `store/priceList/record/created`  | Fires when a price list is created. | 
+| `store/priceList/records/created/` | Fires when price list records are updated. | 
+| `store/priceList/record/updated/` | Fires when a price list record is updated. | 
+| `store/priceList/records/updated/` | Fires when price list records are updated. | 
+| `store/priceList/record/deleted/` | Fires when a price list record is deleted. | 
+
+
+Payload objects with the following scopes take the form that follows:
+
+* `store/pricelist/record/created`
+* `store/pricelist/records/updated/` 
+* `store/pricelist/records/deleted/`
+
+
+```json filename="Example price list records created payload object" showLineNumbers copy
+{
+  "scope": "store/priceList/records/created",
```

Comment:
This is the payload for `priceList/record/created`

The one for multiple records looks like:
<img width="527" alt="Screenshot 2024-03-07 at 9 13 31 AM" src="https://github.com/bigcommerce/docs/assets/1259260/4937f256-5d05-42e5-9c98-18fdb72f7ed1">